<ui version="4.0" >
 <class>NQVTKWindow</class>
 <widget class="QMainWindow" name="NQVTKWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>NQVTK</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>19</y>
     <width>800</width>
     <height>662</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" >
    <property name="spacing" >
     <number>0</number>
    </property>
    <property name="margin" >
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="NQVTKWidget" native="1" name="nqvtkwidget" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>3</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy" >
        <enum>Qt::StrongFocus</enum>
       </property>
      </widget>
      <widget class="QFrame" name="simpleViewFrame" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
         <horstretch>0</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape" >
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Raised</enum>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Expanding" hsizetype="Minimum" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="frameShape" >
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Raised</enum>
      </property>
      <property name="horizontalScrollBarPolicy" >
       <enum>Qt::ScrollBarAlwaysOff</enum>
      </property>
      <property name="widgetResizable" >
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents" >
       <property name="geometry" >
        <rect>
         <x>0</x>
         <y>0</y>
         <width>170</width>
         <height>660</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3" >
        <item>
         <widget class="QGroupBox" name="generalGroup" >
          <property name="title" >
           <string>General</string>
          </property>
          <layout class="QVBoxLayout" name="_2" >
           <property name="spacing" >
            <number>6</number>
           </property>
           <property name="margin" >
            <number>9</number>
           </property>
           <item>
            <widget class="QLabel" name="maxLayersLabel" >
             <property name="text" >
              <string>maxLayers</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="maxLayers" >
             <property name="minimum" >
              <number>1</number>
             </property>
             <property name="maximum" >
              <number>50</number>
             </property>
             <property name="value" >
              <number>8</number>
             </property>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="opacityLabel" >
             <property name="text" >
              <string>opacity</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="opacity" >
             <property name="maximum" >
              <number>100</number>
             </property>
             <property name="value" >
              <number>30</number>
             </property>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="eyeSpacingLabel" >
             <property name="text" >
              <string>eyeSpacing</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="eyeSpacing" >
             <property name="maximum" >
              <number>1000</number>
             </property>
             <property name="value" >
              <number>100</number>
             </property>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="ibisGroup" >
          <property name="title" >
           <string>IBIS</string>
          </property>
          <property name="checkable" >
           <bool>false</bool>
          </property>
          <layout class="QVBoxLayout" name="_3" >
           <property name="spacing" >
            <number>6</number>
           </property>
           <property name="margin" >
            <number>9</number>
           </property>
           <item>
            <widget class="QCheckBox" name="useGridTexture" >
             <property name="text" >
              <string>useGridTexture</string>
             </property>
             <property name="checked" >
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="useGlyphTexture" >
             <property name="text" >
              <string>useGlyphTexture</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="contourDepthEpsilonLabel" >
             <property name="text" >
              <string>contourDepthEpsilon</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="contourDepthEpsilon" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="layoutDirection" >
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="maximum" >
              <number>400</number>
             </property>
             <property name="value" >
              <number>5</number>
             </property>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="useFatContours" >
             <property name="text" >
              <string>useFatContours</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="depthCueRangeLabel" >
             <property name="text" >
              <string>depthCueRange</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="depthCueRange" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimum" >
              <number>1</number>
             </property>
             <property name="maximum" >
              <number>30000</number>
             </property>
             <property name="value" >
              <number>10000</number>
             </property>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="useFog" >
             <property name="text" >
              <string>useFog</string>
             </property>
             <property name="checked" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="distanceFieldsGroup" >
          <property name="title" >
           <string>DistanceFields</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2" >
           <item>
            <widget class="QLabel" name="classificationThresholdLabel" >
             <property name="text" >
              <string>classificationThreshold</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="classificationThreshold" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximum" >
              <number>1500</number>
             </property>
             <property name="value" >
              <number>0</number>
             </property>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="useDistanceColorMap" >
             <property name="text" >
              <string>useDistanceColorMap</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="lightingGroup" >
          <property name="title" >
           <string>Lighting</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout" >
           <item>
            <widget class="QDial" name="lightOffsetDirection" >
             <property name="maximumSize" >
              <size>
               <width>16777215</width>
               <height>50</height>
              </size>
             </property>
             <property name="maximum" >
              <number>360</number>
             </property>
             <property name="singleStep" >
              <number>5</number>
             </property>
             <property name="pageStep" >
              <number>15</number>
             </property>
             <property name="value" >
              <number>270</number>
             </property>
             <property name="wrapping" >
              <bool>true</bool>
             </property>
             <property name="notchTarget" >
              <double>2.000000000000000</double>
             </property>
             <property name="notchesVisible" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="lightRelativeToCamera" >
             <property name="text" >
              <string>lightRelativeToCamera</string>
             </property>
             <property name="checked" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer" >
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>681</y>
     <width>800</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NQVTKWidget</class>
   <extends>QWidget</extends>
   <header>Widgets/NQVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
